system_tests/caff_node_test.go

```diff
@@ -23,23 +23,21 @@ func createCaffNode(ctx context.Context, t *testing.T, existing *NodeBuilder) (*
 	nodeConfig.Sequencer = false
 	nodeConfig.Dangerous.NoSequencerCoordinator = true
 	execConfig.Sequencer.Enable = false
-	execConfig.Sequencer.EnableCaffNode = true
 	execConfig.ForwardingTarget = existing.l2StackConfig.IPCPath
 	execConfig.SecondaryForwardingTarget = []string{}
-	execConfig.Sequencer.CaffNodeConfig.Namespace = builder.chainConfig.ChainID.Uint64()
-	execConfig.Sequencer.CaffNodeConfig.NextHotshotBlock = 1
-	execConfig.Sequencer.CaffNodeConfig.ParentChainNodeUrl = "http://0.0.0.0:8545"
-	execConfig.Sequencer.CaffNodeConfig.EspressoTEEVerifierAddr = existing.L1Info.GetAddress("EspressoTEEVerifierMock").Hex()
-	execConfig.Sequencer.CaffNodeConfig.ParentChainReader.Enable = true
-	execConfig.Sequencer.CaffNodeConfig.ParentChainReader.UseFinalityData = true
-	execConfig.Sequencer.CaffNodeConfig.RecordPerformance = true
+	nodeConfig.EspressoCaffNode.Enable = true
+	nodeConfig.EspressoCaffNode.Namespace = builder.chainConfig.ChainID.Uint64()
+	nodeConfig.EspressoCaffNode.NextHotshotBlock = 1
+	nodeConfig.EspressoCaffNode.EspressoTEEVerifierAddr = existing.L1Info.GetAddress("EspressoTEEVerifierMock").Hex()
+
 	// for testing, we can use the same hotshot url for both
-	execConfig.Sequencer.CaffNodeConfig.HotShotUrls = []string{hotShotUrl, hotShotUrl, hotShotUrl, hotShotUrl}
-	execConfig.Sequencer.CaffNodeConfig.RetryTime = time.Second * 1
-	execConfig.Sequencer.CaffNodeConfig.HotshotPollingInterval = time.Millisecond * 100
-	nodeConfig.ParentChainReader.Enable = false
+	nodeConfig.EspressoCaffNode.HotShotUrls = []string{hotShotUrl, hotShotUrl, hotShotUrl, hotShotUrl}
```

Comment:
Btw, why 4 haha?

Reply:
I don't think there is any particular reason, we just want to have enough for the multiple nodes client to work correctly.